bigquery-storage/api/BigQueryStorage.Samples.Tests/AppendRowsPendingTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Google.Api.Gax;
+using Google.Apis.Bigquery.v2.Data;
+using Google.Cloud.BigQuery.V2;
+using Xunit;
+
+[Collection(nameof(BigQueryStorageFixture))]
+public class AppendRowsPendingTests
+{
+    private readonly BigQueryStorageFixture _fixture;
+
+    private readonly AppendRowsPendingSample _sample;
+
+    private Dictionary<long, (long, string)> _testData;
```

Comment:
Separate note: if you give the value tuple element names, it would be clearer what's expected.